client/client-multi/client-cpp/kaa/IKaaClient.hpp

```diff
@@ -17,7 +17,19 @@
 #ifndef IKAACLIENT_HPP_
 #define IKAACLIENT_HPP_
 
-#include "kaa/security/KeyUtils.hpp"
+#include "kaa/ClientStatus.hpp"
```

Comment:
Do we really need all these inclusions? I think we can replace a lot of them with forward declarations.